README.md

```diff
@@ -89,6 +89,7 @@ It's copy of our example file `config.json.sample`. More or less it looks like:
 |               |      prefix     |  String | Prepend filename prefix if supplied.                                                                                                      |
 |               |      suffix     |  String | Append filename suffix if supplied.                                                                                                       |
 |               |       acl       |  String | Permission of S3 object. [See AWS ACL documentation](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#putObject-property).  |
+|               |       move      | Boolean | If true, an original uploaded file will delete from Bucket after completion.                                                              |
```

Comment:
I guess it should be `true` instead of true, to keep the styling similar to the rest.